AirCasting/SessionsSynchronization/Controller/SessionSynchronizationController.swift

```diff
@@ -26,22 +28,23 @@ final class SessionSynchronizationController: SessionSynchronizer {
     init(synchronizationContextProvider: SessionSynchronizationContextProvidable,
          downstream: SessionDownstream,
          upstream: SessionUpstream,
-         store: SessionSynchronizationStore) {
+         store: SessionSynchronizationStore,
+         viewModel: SessionSynchronizationViewModel) {
```

Comment:
leaving aside the problem that it should not be here in the first place - you should be using a protocol type here, not concrete class (why even create protocols if we inject concrete objects?)

Reply:
My bad, I was doing this and the idea was to use the protocol. Something has happened, I cannot explain I need to say